remo/sdk.py

```diff
@@ -762,3 +762,45 @@ def view_annotation_stats(self, annotation_set_id: int):
 
     def _view(self, url, *args, **kwargs):
         return self.viewer.browse(self.api.url(url, *args, **kwargs))
+    
+    def generate_annotations_from_folders(path_to_data_folder: str):
+        """
+        Creates a CSV annotation file for image classification tasks, where images are stored in folders with names matching the labels of the images. The CSV file is saved in the same input directory where images are stored. 
+        Example of data structure for a dog / cat dataset: 
+              - cats_and_dogs
+                  - dog
+                     - img1.jpg
+                     - img2.jpg
+                     - ...
+                  - cat
+                     - img199.jpg
+                     - img200.jpg
+                     - ...
+            
+        Example::
+            # Unzip the file provided in the the following URL
+            # URL: s-3.s3-eu-west-1.amazonaws.com/cats_and_dogs.zip
+            data_path = "dogs_vs_cats"
+            remo.generate_annotations_from_folders(path_to_data_folder=data_path)
+            
+        Args: 
+               path_to_data_folder: path to the source folder where data is stored
+               class_encoding: path to class_encoding file
```

Comment:
@SreeHarshaNelaturu don't forget to remove `class_encoding` from Args list in doc string

Reply:
@SreeHarshaNelaturu this is still pending i think

Reply:
Yup -- thought I had pushed it from local. done.